--- a/webview/assets/app-initial-236e1501144c.js
+++ b/webview/assets/app-initial-236e1501144c.js
@@ -48761,7 +48761,8 @@
           handler: (e, t) => {
             let { nativeEvent: n } = e,
               { onActivation: r } = t;
-            return !n.isPrimary || n.button !== 0
+            return !n.isPrimary || n.button !== 0 ||
+              (n.pointerType === `touch` && n.target instanceof Element && n.target.closest(`.app-shell-left-panel`))
               ? !1
               : (r?.({ event: n }), !0);
           },
